src/media/js/site/containers/tos.js

```diff
@@ -4,6 +4,22 @@ import {bindActionCreators} from 'redux';
 import urlJoin from 'url-join';
```

Comment:
you could probably re-use this container for the standalone view. just need two conditionals whether the user has signed it, and if so, display the buttons and stuff.